lib/src/authorization_code_grant.dart

```diff
@@ -148,7 +156,7 @@ class AuthorizationCodeGrant {
     };
 
     if (state != null) parameters['state'] = state;
-    if (!scopes.isEmpty) parameters['scope'] = scopes.join(' ');
+    if (!scopes.isEmpty) parameters['scope'] = scopes.join(_delimiter ??= ' ');
```

Comment:
The `??` shouldn't be necessary here, since `_delimiter` is always non-null now.